hs_core/search_indexes.py

```diff
@@ -1044,7 +1045,7 @@ def prepare_aggregation_statistics(self, obj):
         Return list of aggregation statistics if exists, otherwise return empty array.
         """
         aggregation_statistics = []
-        if hasattr(obj, 'metadata'):
+        if hasattr(obj, 'metadata') and obj.metadata is not None:
```

Comment:
These are necessary changes to avoid crashes when reading metadata, but I thought I did these already. I am surprised that they are showing up as changes for this PR.

Reply:
Shouldn't the line below evaluate the same.  If obj.metadata is None then it is not an instance of TimeSeriesMetaData.